src/gcode-streamer_test.cc

```diff
@@ -0,0 +1,190 @@
+/* -*- mode: c++; c-basic-offset: 2; indent-tabs-mode: nil; -*-
+ * Test for pru motion queue.
+ *
+ * We simulate the pru hardware and test if the absolute position is correctly
```

Comment:
This header comment looks like it has been copied from somewhere else :)

Reply:
(solved) *facepalm*